NDB.Covid19/NDB.Covid19.iOS/Views/DailyNumbers/DailyNumbers.storyboard

```diff
@@ -445,7 +445,7 @@
                                                                     </view>
                                                                 </subviews>
                                                                 <constraints>
-                                                                    <constraint firstItem="WRL-i3-xEC" firstAttribute="firstBaseline" secondItem="ubU-j6-5GM" secondAttribute="firstBaseline" id="lB7-hn-xDY"/>
+                                                                    <constraint firstItem="WRL-i3-xEC" firstAttribute="centerY" secondItem="Meh-Oj-gTx" secondAttribute="centerY" id="hUf-xN-k29"/>
```

Comment:
All other boxes use `firstBaseline` for this sort of alignment, please do the same here (instead of `centerY`) so it is consistent with the rest of the storyboard.

Reply:
_won't be fixed_